out/_next/static/chunks/pages/form-a2948814b229a3f2.js

```
@@ -0,0 +1 @@
+(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[410],{7253:(t,e,a)=>{(window.__NEXT_P=window.__NEXT_P||[]).push(["/form",function(){return a(7395)}])},7395:(t,e,a)=>{"use strict";a.r(e),a.d(e,{default:()=>s});var i=a(4232),r=a(5986),n=a(7328),l=a.n(n),d=a(4323);function s(){return(0,r.FD)(i.Fragment,{children:[(0,r.FD)(l(),{children:[(0,r.Y)("meta",{name:"viewport",content:"width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"}),(0,r.Y)("title",{children:"Want to be Part of HackClub Butwal"}),(0,r.Y)("script",{async:!0,src:"https://tally.so/widgets/embed.js"})]}),(0,r.Y)(d.az,{sx:{position:"fixed",top:0,left:0,right:0,bottom:0,height:"100vh",width:"100vw",overflow:"hidden",margin:0,padding:0},children:(0,r.Y)("iframe",{"data-tally-src":"https://tally.so/r/mO9a6K?transparentBackground=1",width:"100%",height:"100%",frameBorder:"0",marginHeight:"0",marginWidth:"0",title:"Want to be Part of HackClub Butwal",sx:{border:0,display:"block"}})})]})}}},t=>{var e=e=>t(t.s=e);t.O(0,[323,636,593,792],()=>e(7253)),_N_E=t.O()}]);
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)